NvAPIWrapper/Native/DisplayApi.cs

```diff
@@ -211,6 +211,34 @@ public static UnAttachedDisplayHandle GetAssociatedUnAttachedNvidiaDisplayHandle
             return display;
         }
 
+        /// <summary>
+        ///     This function returns color information relating to the targeted display id.
+        /// </summary>
+        /// <param name="displayId">The targeted display output id.</param>
+        /// <returns>Data corresponding to color information</returns>
+        /// <exception cref="NVIDIAApiException">Status.Error: Miscellaneous error occurred</exception>
+        /// <exception cref="Exception">A delegate callback throws an exception.</exception>
+        public static TColorData GetColorData<TColorData>(uint displayId) where TColorData : struct, IColorData
+        {
+            var colorData = (TColorData)Activator.CreateInstance(typeof(TColorData), ColorCommand.Get);
```

Comment:
Probably should be changed to the following so it feels inline with other methods in the library:

```C#
var colorData = typeof(TColorData).Instantiate<TColorData>();
colorData.ColorCommand = ColorCommand.Get;
```